resources/queries/targetedms/peptidegroup/LibraryProteins.qview.xml

```diff
@@ -10,7 +10,7 @@
         <column name="RepresentativeDataState" />
     </columns>
     <filters>
-        <filter column="RepresentativeDataState" operator="notin" value="2"/>
+        <filter column="RepresentativeDataState" operator="eq" value="1"/>
```

Comment:
@labkey-jeckels This change broke the TargetedMSMxNReproducibilityReportTest. The test creates a peptide library folder and was looking for "Reproducibility Reports" etc. links in the "Proteins" tab.  In a peptide library folder, proteins are not marked as representative, so the "Library Proteins" view displayed in the "Proteins" tab was empty since no rows matched the filter.  I have updated the PeptideGroupViewWebpart to show the "Library Proteins" view only in protein library folder. Other folders will show the default view.